db/db.js

```diff
@@ -245,12 +245,12 @@ async function updateSessionState(sessionid, state, clientParam) {
 }
 
 // Updates the value of the alert flag in the location database
-async function updateSentAlerts(location, sentalerts, clientParam) {
+async function updateSentAlerts(locationid, sentalerts, clientParam) {
   try {
     const results = await runQuery(
       'updateSentAlerts',
       'UPDATE locations SET heartbeat_sent_alerts = $1 WHERE locationid = $2 RETURNING *',
-      [sentalerts, location],
+      [sentalerts, locationid],
```

Comment:
Changed this variable name to make it clearer that what we're passing is just the Location's ID, not the whole Location object.